File: 2026-09/src/day_13/leet_268/Solution3.java
package day_13.leet_268;

class Solution3 {
    public int missingNumber(int[] nums) {
        int ans = 0;
        for (int i = 1; i <= nums.length; i++) {
            ans ^= i;
        }
        for (int x : nums) {
            ans ^= x;
        }

        return ans;
    }
}

File: 2026-09/src/day_13/leet_268/Solution3.cpp
#include <bits/stdc++.h>
using namespace std;

class Solution {
   public:
    int missingNumber(vector<int>& nums) {
        int ans = 0;
        for (int i = 1; i <= nums.size(); i++) ans ^= i;
        for (int x : nums) ans ^= x;

        return ans;
    }
};
